tsweb/varz/varz_test.go

```diff
@@ -418,3 +420,65 @@ func TestVarzHandlerSorting(t *testing.T) {
 		}
 	}
 }
+
+func TestWriteMemestats(t *testing.T) {
+	var w bytes.Buffer
+	writeMemstats(&w, &runtime.MemStats{
+		Alloc:        1,
+		TotalAlloc:   2,
+		Sys:          3,
+		Lookups:      4,
+		Mallocs:      5,
+		Frees:        6,
+		HeapAlloc:    7,
+		HeapSys:      8,
+		HeapIdle:     9,
+		HeapInuse:    10,
+		HeapReleased: 11,
+		HeapObjects:  12,
+		StackInuse:   13,
+		StackSys:     14,
+		MSpanInuse:   15,
+		MSpanSys:     16,
+		MCacheInuse:  17,
+		MCacheSys:    18,
+		BuckHashSys:  19,
+		GCSys:        20,
+		OtherSys:     21,
+		NextGC:       22,
+		LastGC:       23,
+		PauseTotalNs: 24,
+		// PauseNs:       [256]int64{},
+		NumGC:         26,
+		NumForcedGC:   27,
+		GCCPUFraction: 28.1,
+	})
```

Comment:
pull this memstats out to a variable and then you can add a `testing.AllocsPerRun(1000, func(){ buf.Reset() writeMemstats(&buf, ms)` and verify it doesn't allocate